opentracing-api/pom.xml

```diff
@@ -29,8 +29,8 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
-        <main.java.version>1.6</main.java.version>
-        <main.signature.artifact>java16</main.signature.artifact>
+        <main.java.version>1.7</main.java.version>
```

Comment:
Due to [plumbr](https://plumbr.eu/blog/java/java-version-and-vendor-data-analyzed-2016-edition), almost 10% of their customers have used Java 1.6 in 2016.

I think a big milestone for us framework developers will be the release of Spring 5 this year which only supports Java 8+. So my hope is that within a year or two, it will be "mainstream" to drop support for Java 6 and maybe even 7 in libraries like OpenTracing. Especially since this is a API and not an implementation it is imho important to support as many Java versions as possible.